### ISSUE TITLE
[bottles (2022.10.14.1)] error when creating a bottle

### ISSUE THREAD

**Issue by MurdeRM3L0DY** [NixOS/nixpkgs#203051]
### Describe the bug
error when creating a bottle

### Steps To Reproduce
1. launch bottles
2. try to create a new bottle
3. error during dxvk installation process with the following error message
```console
13:21:02 (INFO) Bottles Started! 
13:21:03 (INFO) Catalog components loaded 
13:21:03 (INFO) Catalog installers loaded 
13:21:04 (INFO) Catalog dependencies loaded 
13:21:04 (INFO) Performing Bottles checks… 
13:21:04 (INFO) Dxvks found:
	- dxvk-2.0
 
13:21:04 (INFO) Vkd3ds found:
	- vkd3d-proton-2.7
 
13:21:04 (INFO) Nvapis found:
	- dxvk-nvapi-v0.5.4
 
13:21:04 (INFO) Latencyflexs found:
	- latencyflex-v0.1.1
 
13:21:04 (INFO) Runners found:
	- soda-7.0-7
 
13:21:04 (INFO) Library saved 
13:21:17 (INFO) Generating bottle configuration… 
13:21:17 (INFO) Using Wine Runtime tool -- send_status(-i) 
13:21:17 (WARNING) No GPU vendor found, keep going without setting VK_ICD_FILENAMES… 
wineserver: using server-side synchronization.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
wine: configuration in L"/home/nemesis/.local/share/bottles/bottles/flstudio__145" has been updated.
13:21:22 (INFO) Setting Windows version… 
13:21:22 (INFO) Setting CMD default settings… 
13:21:22 (INFO) Importing bundle to flstudio registry 
13:21:22 (INFO) Using Wine Registry CLI -- import_bundle 
13:21:22 (WARNING) No GPU vendor found, keep going without setting VK_ICD_FILENAMES… 
wineserver: using server-side synchronization.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -r" (2).
13:21:23 (INFO)  
13:21:23 (INFO) Using Wine Runtime tool -- send_status(-u) 
13:21:23 (WARNING) No GPU vendor found, keep going without setting VK_ICD_FILENAMES… 
wineserver: using server-side synchronization.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
wine: configuration in L"/home/nemesis/.local/share/bottles/bottles/flstudio__145" has been updated.
13:21:28 (INFO) Optimizing environment… 
13:21:28 (INFO) Adding Key: [HKEY_CURRENT_USER\Software\Wine\DllOverrides] with Value: [winemenubuilder.exe] and Data: [] in flstudio registry 
13:21:28 (INFO) Using Wine Registry CLI -- add 
13:21:28 (WARNING) No GPU vendor found, keep going without setting VK_ICD_FILENAMES… 
wineserver: using server-side synchronization.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -r" (2).
13:21:29 (INFO) reg: The operation completed successfully
	
 
13:21:29 (INFO) Adding Key: [HKEY_CURRENT_USER\Software\Wine\DllOverrides] with Value: [mshtml] and Data: [] in flstudio registry 
13:21:29 (INFO) Using Wine Registry CLI -- add 
13:21:29 (WARNING) No GPU vendor found, keep going without setting VK_ICD_FILENAMES… 
wineserver: using server-side synchronization.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -r" (2).
13:21:30 (INFO) reg: The operation completed successfully
	
 
13:21:30 (INFO) Applying environment: [application]… 
13:21:30 (INFO) Installing DXVK… 
13:21:30 (ERROR) Error while running async job: <bound method Manager.create_bottle of <bottles.backend.managers.manager.Manager object at 0x7fd43cc82740>>
	Exception: 'x32'
 
  File "/nix/store/rar8iqlij122flqs290589iyy88z0qxl-bottles-2022.10.14.1/share/bottles/bottles/frontend/utils/threading.py", line 63, in __target
    result = self.task_func(*args, **kwargs)
  File "/nix/store/rar8iqlij122flqs290589iyy88z0qxl-bottles-2022.10.14.1/share/bottles/bottles/backend/managers/manager.py", line 1318, in create_bottle
    self.install_dll_component(config, "dxvk", version=dxvk_name)
  File "/nix/store/rar8iqlij122flqs290589iyy88z0qxl-bottles-2022.10.14.1/share/bottles/bottles/backend/managers/manager.py", line 1509, in install_dll_component
    manager = DXVKComponent(_version)
  File "/nix/store/rar8iqlij122flqs290589iyy88z0qxl-bottles-2022.10.14.1/share/bottles/bottles/backend/dlls/dll.py", line 39, in __init__
    self.check()
  File "/nix/store/rar8iqlij122flqs290589iyy88z0qxl-bottles-2022.10.14.1/share/bottles/bottles/backend/dlls/dll.py", line 57, in check
    del found[path][dll]
```

### Expected behavior
No error when trying to create a bottle

### Screenshots
![2022-11-26_13-22](https://user-images.githubusercontent.com/95099980/204088750-db4f2560-ef3c-415b-80b1-e984ec317f64.png)

### Additional context
I'm pretty new to nixos so it might be a configuration error on my part

### Notify maintainers
@psydvl @SCOTT-HAMILTON @SuperSamus
I apologize in advanced if I tagged the wrong maintainers

### Metadata
```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.0.9, NixOS, 23.05 (Stoat), 23.05.20221124.27ccd29`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.0`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by SuperSamus** [comment]
It should be fixed in #202265 (I don't reproduce it there).
(I think it's caused by the fact I [forgot a comma](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/misc/bottles/vulkan_icd.patch))

**Comment by MurdeRM3L0DY** [comment]
gotcha. feel free to close if you think it's gonna be fixed in next release. thanks :)
